code/datums/keybinding/mob.dm

```diff
@@ -57,7 +57,7 @@
 
 /datum/keybinding/mob/stop_pulling/down(client/user)
 	var/mob/M = user.mob
-	if (M.pulling)
+	if(!M.pulling)
```

Comment:
```suggestion
	if(M.pulling)
	    M.stop_pulling()
```

Just delete the rest, I don't feel the message is helpful on keybindings.

Reply:
Any chance of making this a signal? I can guide you through how, and it's useful knowledge.

Reply:
reeee sure

Reply:
doesnt need to be a signal, pulling and stop_pulling are defined at mob or above

Reply:
just do the suggested change and it can merge